src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.test.js

```diff
@@ -31,190 +31,62 @@ describe('<List />', () => {
-            state: 'm',
-            state_display: 'Gemeld',
-            extra_properties: {},
-            created_at: '2018-11-29T23:03:19.566082+01:00',
-          },
-          location: {
-            id: 1637,
-            stadsdeel: 'K',
-            buurt_code: 'A04h',
-            address: {
-              postcode: '1011KJ',
-              huisletter: '',
-              huisnummer: '45',
-              woonplaats: 'Amsterdam',
-              openbare_ruimte: 'Raamgracht',
-              huisnummer_toevoeging: '',
-            },
-            address_text: 'Raamgracht 45 1011KJ Amsterdam',
-            geometrie: {
-              type: 'Point',
-              coordinates: [
-                4.900460243225099,
-                52.3692814746251,
-              ],
-            },
-            extra_properties: null,
-          },
-          incident_date_end: null,
-          updated_at: '2018-11-29T23:05:52.590923+01:00',
-          _links: {
-            self: {
-              href: 'https://acc.api.data.amsterdam.nl/signals/auth/signal/1667/',
-            },
-          },
-          source: 'Telefoon – CCA',
-          id: 1667,
-          image: null,
-          operational_date: null,
-          category: {
-            sub: 'Dode dieren',
-            sub_slug: 'dode-dieren',
-            main: 'Overlast van dieren',
-            main_slug: 'overlast-van-dieren',
-            department: 'ASC, CCA, GGD',
-            category_url: 'https://category-url.amsterdam.nl',
-          },
-          incident_date_start: '2018-11-29T23:03:19+01:00',
-          text_extra: '',
-        },
-      ],
-      priorityList,
-      statusList,
-      stadsdeelList,
-      incidentSelected: jest.fn(),
+      incidents,
+      priority: priorityList,
+      status: statusList,
+      stadsdeel: stadsdeelList,
       onChangeOrdering: jest.fn(),
     };
-
-    wrapper = shallow(<List {...props} />);
   });
 
   afterEach(() => {
     jest.resetAllMocks();
   });
 
   it('should render correctly', () => {
-    expect(wrapper).toMatchSnapshot();
+    const { container } = render(withAppContext(<List {...props} />));
+
+    expect(container.querySelector('tr th:nth-child(1)')).toHaveTextContent(/^Id$/)
+    expect(container.querySelector('tr th:nth-child(2)')).toHaveTextContent(/^Dag$/)
+    expect(container.querySelector('tr th:nth-child(3)')).toHaveTextContent(/^Datum en tijd$/)
+    expect(container.querySelector('tr th:nth-child(4)')).toHaveTextContent(/^Stadsdeel$/)
+    expect(container.querySelector('tr th:nth-child(5)')).toHaveTextContent(/^Subcategorie$/)
+    expect(container.querySelector('tr th:nth-child(6)')).toHaveTextContent(/^Status$/)
+    expect(container.querySelector('tr th:nth-child(7)')).toHaveTextContent(/^Urgentie$/)
+    expect(container.querySelector('tr th:nth-child(8)')).toHaveTextContent(/^Adres$/);
+
+
+    expect(container.querySelector('tr:nth-child(1) td:nth-child(1)')).toHaveTextContent(/^1668$/)
```

Comment:
Can you query the values from the fixture instead of hardcoding them?

Reply:
dat kan niet het want er zitten codes die worden vertaald in voor stadsdeel, urgentie en status. en ook de datum wordt geformat.

ik heb nu alle velden die wel werken erin gezet